integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/ui/portal/TransferFragment.kt

```diff
@@ -339,30 +339,39 @@ class TransferFragment : Fragment(R.layout.fragment_transfer) {
     }
 
     private suspend fun showWithdrawalLimitsError(period: WithdrawalLimitPeriod) {
-        val limits = viewModel.getWithdrawalLimits()
-        val okButtonText = if (period == WithdrawalLimitPeriod.PerTransaction) {
-            if (viewModel.onlineAccountStatus == OnlineAccountStatus.Done) {
-                getString(R.string.read_withdrawal_policy)
+        if (period == WithdrawalLimitPeriod.PerBlock) {
+            AdaptiveDialog.create(
+                R.drawable.ic_warning,
+                "Please wait before initiating the next withdrawal",
+                "Please wait 5 minutes before initiating another withdrawal",
+                getString(R.string.button_okay)
+            ).showAsync(requireActivity())
+        } else {
+            val limits = viewModel.getWithdrawalLimits()
+            val okButtonText = if (period == WithdrawalLimitPeriod.PerTransaction) {
+                if (viewModel.onlineAccountStatus == OnlineAccountStatus.Done) {
+                    getString(R.string.read_withdrawal_policy)
+                } else {
+                    getString(R.string.online_account_create)
+                }
             } else {
-                getString(R.string.online_account_create)
+                ""
             }
```

Comment:
if the period is PerBlock, then show one dialog, otherwise show the regular limit dialog